packages/jest-haste-map/src/lib/FSEventsWatcher.ts

```diff
@@ -11,7 +11,6 @@ import path from 'path';
 import {EventEmitter} from 'events';
 import anymatch from 'anymatch';
 import micromatch from 'micromatch';
-// eslint-disable-next-line
```

Comment:
This will probably break CI :/ Kind of annoying to get these lint errors locally I know, see [here](https://github.com/facebook/jest/pull/8311#issuecomment-498863636) for an explanation

Reply:
Oh... I see.
Let me add this line back